packages/unmock-core/src/service/interfaces.ts

```diff
@@ -0,0 +1,22 @@
+export type IOASMappingGenerator = () => { [key: string]: OASSchema };
+
+export interface IStateMapping {
+  [key: string]: any;
+}
+
+// Type aliases for brevity
+export type OASSchema = any;
```

Comment:
This does not use `openapi3-ts` because Mike's writing the types for that, is that the reason?

Reply:
Pretty much - it's a place holder for now following your suggestion [here](https://github.com/unmock/unmock-js/pull/50#discussion_r295165222).

Same for the below comments.

Reply:
It's fine to have a place-holder, just asking why use `any` if a type would be available :)

Reply:
And `[key: string]: any` is still different to me than `[serviceName: string]: Specification` even if both passed the same type tests.

Reply:
Ah! Fair enough. Will update those to reflect their connection with one another.